core/test/unit/web/middleware/image/handle-image-sizes_spec.js

```diff
@@ -0,0 +1,43 @@
+const should = require('should');
+const handleImageSizes = require('../../../../../server/web/shared/middlewares/image/handle-image-sizes.js');
+
+describe('handleImageSizes middleware', function () {
+    it('calls next immediately if the url does not match /size/something/', function (done) {
+        const fakeReq = {
+            url: '/size/something'
+        };
+        // CASE: second thing middleware does is try to match to a regex
```

Comment:
These tests are rubbish because they're _highly_ tied to the implementation. in fact they deserve to have a `TODO` - will add now

Reply:
Agreed on the rubbish point :smile:  But this gives us an opportunity to open an issue for improvement, so somebody from the community could pick it up :wink: